data/categories.txt

```diff
@@ -7876,9 +7878,23 @@ el:Δικαστικό μέγαρο
 sk:Súd
 fa:دادگاه
 
+amenity-vending_machine
```

Comment:
If we want to have common type amenity=vending_machine, we should manually declare it in mapcss-mappning.csv

Do we want?

Reply:
We can add additional "excrement_bags" and "parcel_pickup", w/o common type (who needs it?)

Reply:
yes please